NuGet.Config

```diff
@@ -10,6 +10,7 @@
       Used for azure-sdk-tools repo until issue https://github.com/Azure/azure-sdk-tools/issues/1329 is addressed
     -->
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
+    <add key="myget" value="https://www.myget.org/F/azure-appservice/api/v3/index.json" />
```

Comment:
This PR cannot merge with this new nuget feed included.

Reply:
I blocked the merge until the necessary package has been released and this can be removed.